pkgs/by-name/re/reaction/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitLab,
+}:
+let
+  pname = "reaction";
+  version = "1.1.2";
+
+  src = fetchFromGitLab {
+    domain = "framagit.org";
+    owner = "ppom";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-3YG0MWy0Ocb8Jn6QxM3M+gZm4UWfTjwo/Y6cwEkceHc=";
+  };
+
+  ip46tables = stdenv.mkDerivation {
+    inherit version src;
+    pname = "ip46tables";
+    buildPhase = ''
+      gcc ip46tables.d/ip46tables.c -o ip46tables
+    '';
```

Could this derivation be inlined into the `buildGoModule` derivation in  `postBuild` and `postInstall`? 

If not
```suggestion
    buildPhase = ''
      runHook preBuild
      gcc ip46tables.d/ip46tables.c -o ip46tables
      runHook postBuild
    '';
```
and the same for `installPhase`

---

Nice idea, thanks